src/spk-client.ts

```diff
@@ -97,6 +97,7 @@ export class SpkClient {
       {
         parent_id: parentId,
         content,
+        created_at: new Date(),
```

Comment:
Convert to ISOString (if it does not do it already). Good to be more explicit on conversions.